Microsoft.Toolkit.Graph.Controls/Controls/GraphPresenter/GraphPresenter.cs

```diff
@@ -0,0 +1,113 @@
+        /// Set to the base item type and use the <see cref="IsCollection"/> property to indicate if a collection is expected back.
+        /// </summary>
+        public Type ResponseType { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether the returned data from the <see cref="RequestBuilder"/> is a collection.
+        /// </summary>
+        public bool IsCollection { get; set; }
+
+        /// <summary>
+        /// Gets or sets list of <see cref="QueryOption"/> representing <see cref="Microsoft.Graph.QueryOption"/> values to pass into the request built by <see cref="RequestBuilder"/>.
+        /// </summary>
+        public List<QueryOption> QueryOptions { get; set; } = new List<QueryOption>();
+
+        /// <summary>
+        /// Gets or sets a string to indicate a sorting order for the <see cref="RequestBuilder"/>. This is a helper to add this specific request option to the <see cref="QueryOptions"/>.
+        /// </summary>
+        public string OrderBy { get; set; }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="GraphPresenter"/> class.
+        /// </summary>
+        public GraphPresenter()
+        {
+            Loaded += GraphPresenter_Loaded;
+        }
+
+        private async void GraphPresenter_Loaded(object sender, RoutedEventArgs e)
+        {
+            // Note: some interfaces from the Graph SDK don't implement IBaseRequestBuilder properly, see https://github.com/microsoftgraph/msgraph-sdk-dotnet/issues/722
+            if (RequestBuilder != null)
+            {
+                var request = new BaseRequest(
+                                        RequestBuilder.RequestUrl,
+                                        RequestBuilder.Client); // TODO: Do we need separate Options here?
+                request.Method = "GET";
+                request.QueryOptions = QueryOptions?.Select(option => option.ToQueryOption())?.ToList() ?? new List<Microsoft.Graph.QueryOption>();
+
+                // Handle Special QueryOptions
+                if (!string.IsNullOrWhiteSpace(OrderBy))
+                {
+                    request.QueryOptions.Add(new Microsoft.Graph.QueryOption("$orderby", OrderBy));
+                }
+
+                try
+                {
+                    var response = await request.SendAsync<object>(null, CancellationToken.None).ConfigureAwait(false) as JObject;
+
+                    //// TODO: Deal with paging?
+
+                    var values = response["value"];
+                    object data = null;
+
+                    if (IsCollection)
+                    {
+                        data = values.ToObject(Array.CreateInstance(ResponseType, 0).GetType());
+                    }
+                    else
+                    {
+                        data = values.ToObject(ResponseType);
+                    }
+
+                    _ = DispatcherHelper.ExecuteOnUIThreadAsync(() => Content = data);
```

Comment:
Use the DispatcherQueue, not the old CoreDispatcher.

Reply:
We're not on the 7.0.0 preview package yet. I need to update later.

Reply:
#57